wordpress-mobile/WordPress-Android
# Replace icons with spectrum icons set.

This changes the app icons set to spectrum icons. As of now, this change is meant to stay in place for a month and then come back to the previous set of icons.

| Scenario | | | | |
|---|---|---|---|---|
| hdpi/api24 |  <img width="363" alt="image" src="https://user-images.githubusercontent.com/47797566/169259566-35b2793a-b8ea-4122-a9d9-179144345a37.png"> | <img width="363" alt="image" src="https://user-images.githubusercontent.com/47797566/169259855-4c428e4e-cfcb-4434-8784-c485fad2567c.png"> | <img width="361" alt="image" src="https://user-images.githubusercontent.com/47797566/169266666-0cbf8ff9-1b54-4267-ad03-2e5a8d890bbc.png"> | <img width="361" alt="image" src="https://user-images.githubusercontent.com/47797566/169266581-0c37ef62-c0e9-4e81-99df-9ddec8d5b8e7.png"> |
| xxhdpi/api29 | <img width="363" alt="image" src="https://user-images.githubusercontent.com/47797566/169262147-af3f74ce-b120-40d9-bee9-e1697e90948f.png"> | <img width="363" alt="image" src="https://user-images.githubusercontent.com/47797566/169262326-467da0e6-1bca-4e19-8d3f-9f48ca156fa3.png"> | <img width="363" alt="image" src="https://user-images.githubusercontent.com/47797566/169266228-922104d5-e0fe-4b7e-974d-736d24b6304b.png"> | <img width="363" alt="image" src="https://user-images.githubusercontent.com/47797566/169266161-256a2787-c2d3-4d19-afa0-31ee4578af2b.png"> |

To test:

Note that the CI built apk builds the Jalapeno variant that has its own set of icons. To test this apk you should build vanilla variant!

- Install vanilla variant on low and high resolution simulators (both with Nougat or below API and Oreo or above API, **the more diverse combinations you can test the better** 😄 ).
- Check the icon is the new one as expected.

## Regression Notes
1. Potential unintended areas of impact
None if properly tested on the various resolutions and API levels threshold.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
4. N/A




PR submission checklist:
